d3a_interface/sim_results/aggregate_results.py

```diff
@@ -0,0 +1,198 @@
+                        for ts in time_list:
+                            if ts not in target_ul[timestamp]['unmatched_times'][device]:
+                                target_ul[timestamp]['unmatched_times'][device].append(ts)
+
+                unm_count = 0
+                for _, hours in target_ul[timestamp]['unmatched_times'].items():
+                    unm_count += len(hours)
+
+                    target_ul[timestamp]['unmatched_count'] = unm_count
+
+    @classmethod
+    def _copy_accumulated_unmatched_loads(cls, accumulated_results, current_results, area):
+        """
+        Copies the accumulated results from the market results to the incremental results
+        :param accumulated_results: stores the merged unmatched load results, changes by reference
+        :param current_results: results for the current market, that are used to update the
+        accumulated results
+        :param area: area of the accumulated unmatched loads
+        :return: None
+        """
+        for timestamp, ts_value in current_results[area]['unmatched_loads'].items():
+            if timestamp not in accumulated_results[area]['unmatched_loads']:
+                accumulated_results[area]['unmatched_loads'][timestamp] = deepcopy(ts_value)
+
+    @classmethod
+    def accumulate_current_market_results(cls, accumulated_results, current_results):
+        """
+        Method which starts the merging of the current market unmatched loads with the
+        existing unmatched loads (_unmatched_loads_incremental)
+        :param accumulated_results: return value, stores the merged unmatched load results
+        :param current_results: results for the current market, that are used to update the
+        accumulated results
+        :return: accumulated_results
+        """
+        for base_area, target_results in current_results.items():
+            if base_area not in accumulated_results:
+                accumulated_results[base_area] = deepcopy(target_results)
+            else:
+                cls._merge_base_area_unmatched_loads(
+                    accumulated_results, current_results, base_area
+                )
+        return accumulated_results
+
+
+def merge_unmatched_load_results_to_global(market_ul, global_ul):
+    if not global_ul:
+        global_ul = market_ul
+        return global_ul
+    return UnmatchedLoadsHelpers.accumulate_current_market_results(global_ul, market_ul)
+
+
+def merge_price_energy_day_results_to_global(market_pe, global_pe):
+    if not global_pe:
+        global_pe = market_pe
+        return global_pe
+    for area_uuid in market_pe:
+        if area_uuid not in global_pe:
+            global_pe[area_uuid] = deepcopy(market_pe[area_uuid])
+        else:
+            global_pe[area_uuid]["price-energy-day"].extend(
+                market_pe[area_uuid]["price-energy-day"])
+    return global_pe
+
+
+def merge_device_statistics_results_to_global(market_device, global_device):
+    if not global_device:
```

Comment:
Minor, but:
`if not {}` yields True. This was new to me. Not sure what I think of it considering the readability. 
Also, the name "*_device" was confusing at first. Maybe append a `_dict` everywhere (in order to not append `_statistics` that would be too long I understand.)

Reply:
Regarding the latter comment, I would suggest to introduce `typing` here, and to denote that these are dicts. That way we can have that information and a potential type check (if we decide to use it in the future)